examples/two-step-transfer/index.html

```diff
@@ -99,6 +124,7 @@
 
     <body>
         <button id="connect">Connect</button>
+        <button id="sendInit">Initiate two step transfer contract</button>
         <button id="sendTransfer">Send transfer</button>
```

Comment:
Should we maybe remove this from the example (perhaps put it somewhere else)?

Could do a "donation" example, which would just be a button saying "Donate to me!" and a send transaction 😁

Reply:
Might be out of scope of the PR, but should be a quick fix, if you find it relevant.

Reply:
Hmm, I know it's not two-step-transfer, but I was thinking this would become a board of buttons to trigger every endpoint.

Reply:
And it would be the same for the signMessage I assume?

Reply:
Ok. Might be confusing if you (as an outsider) come looking for examples. If we want a "board of buttons", I would suggest that we instead make an example for that purpose, that isn't mixed with integration with a concrete smart contract.

Can wait for another time though, but maybe we should make an issue for the backlog?